packages/blade/src/components/Button/BaseButton/getBaseButtonStyles.ts

```diff
@@ -0,0 +1,31 @@
+import type { StyledBaseButtonProps } from './StyledBaseButton';
+
+const getBaseButtonStyles = ({
+  color,
+  hoverColor,
+  activeColor,
+}: Omit<StyledBaseButtonProps, 'children' | 'onClick'>): string => `
+  display:flex;
+  flex-direction: row;
+  height: 48px;
+  cursor: pointer;
+  background-color: ${color};
+  border: 2px solid ${color};
+  color: white;
+  border-radius: 2px;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
```

Comment:
use the spacing token?

Reply:
why not?

Reply:
Ignore all the styles, they'll all change. Will take care of all of this in chained PRs. Check the scope of this PR in the description. This is just a dummy implementation